beatmap_collections/templates/beatmap_collections/add_beatmap.html

```diff
@@ -109,6 +109,9 @@ <h2>Suggest Beatmap</h2>
 {% block js %}
 <script src="https://code.jquery.com/jquery-3.6.0.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>
 <script src="{% static "js/live_beatmap_card.js" %}"></script>
-<script src="https://code.jquery.com/jquery.min.js"></script>
+<script
+        src="https://code.jquery.com/jquery-1.11.2.min.js"
+        integrity="sha256-Ls0pXSlb7AYs7evhd+VLnWsZ/AqEHcXBeMZUycz/CcA="
+        crossorigin="anonymous"></script>
 <script src="{% static "js/player.js" %}"></script>
```

Comment:
This line has multiple jQuery. If the lower one is used, can you remove the upper one?

Reply:
I tried and the live demo can use the version that demo player need. Removed.